tech_docs/source/api-docs/v2025.0/objects/trainee.html.md

```diff
@@ -658,4 +658,116 @@ weight: 1
       </ul>
     </dd>
   </div>
+  <div class="govuk-summary-list__row govuk-summary-list__row--no-actions">
+    <dt class="govuk-summary-list__key"><code>withdrawal_reasons</code></dt>
+    <dd class="govuk-summary-list__value">
+      <p class="govuk-body">
+        array of strings
+      </p>
+      <p class="govuk-body">
+        The list of reasons for the trainee’s withdrawal. (read-only)
+      </p>
+      <p class="govuk-body">
+        Possible values:
```

Comment:
Do we need more values? https://github.com/DFE-Digital/register-trainee-teachers/blob/main/config/initializers/withdrawal_reasons.rb#L3